src/Compilers/CSharp/Portable/Symbols/EmbeddedAttributes.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    [Flags]
+    internal enum EmbeddedAttributes
```

Comment:
>EmbeddedAttributes [](start = 18, length = 18)

"Embedd**able**Attributes"?

Reply:
That's a better name, thanks.

---
In reply to: [293947270](https://github.com/dotnet/roslyn/pull/36398#discussion_r293947270) [](ancestors = 293947270)